### PR TITLE
Liquid frontend

### PR DESCRIPTION
**Pull request by zenry**
- [x] Seeds
- [x] Add Liquid Tag, Block and Filter
- [x] Add Services and Commands
- [x] Liquid Render Service
- [x] Controller render helper
- [x] Load blocks in Page

### CODE REVIEW

File: app/controllers/frontend_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+class FrontendController < ApplicationController
+  def show
+    page = Page.find_by(slug: params[:slug])
+
+    render_page page, page: page
```
Comment (zenry) [inline app/controllers/frontend_controller.rb]: Ugly code, I was super sleepy and this makes no sense, I will change this

//cc @jeroenvisser101

Comment (jeroenvisser101) [inline app/controllers/frontend_controller.rb]: 👍

File: Gemfile [hunk 2]
```diff
@@ -26,14 +26,13 @@ end
 
 group :development, :test do
   gem "byebug"
-
   gem "fuubar"
 
-  gem "simplecov"
+  gem "simplecov", require: false
```
Comment (jeroenvisser101) [inline Gemfile]: good call! :)

File: Gemfile [hunk 3]
```diff
@@ -26,14 +26,13 @@ end
...
   gem "rspec", "~> 3.5.0.beta1", "< 3.6", require: false
   gem "rspec-core", "~> 3.5.0.beta1", "< 3.6", require: false
   gem "rspec-rails", "~> 3.5.0.beta1", "< 3.6", require: false
-  gem "shoulda-matchers", "~> 3.1"
+  gem "shoulda-matchers", "~> 3.1", require: false
```
Comment (jeroenvisser101) [inline Gemfile]: I should probably have that too

**Comment by zenry** [comment]
Can you give this a spin @jeroenvisser101 ?

**Comment by jeroenvisser101** [comment]
Sure I can :)

**Comment by jeroenvisser101** [comment]
Do I have to have a site/domain set up for this to work? I get  'No active site on localhost'

**Comment by zenry** [comment]
https://github.com/uberous/sitebake/blob/liquid-frontend/db/seeds.rb#L14

**Comment by zenry** [comment]
Blocks should also work in Pages now

Migrations and seed have been changed, rebuild db

/cc @jeroenvisser101

**Comment by zenry** [comment]
Gonna merge this to test out RethinkDB

**Pull request merged**